packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -1037,12 +1049,15 @@ const _BaseInput: React.ForwardRefRenderFunction<BladeElementRef, BaseInputProps
               validationState={validationState}
               trailingButton={trailingButton}
               size={size}
+              errorText={errorText}
+              successText={successText}
+              hasBorder={hasBorder}
             />
           </BaseInputWrapper>
         </FocusRingWrapper>
       </BaseBox>
 
-      {!hideFormHint && (
+      {!hideFormHint && hasBorder && (
```

Comment:
should we call this variable `isTableInputCell` or something? a bit confusing that `hasBorder` flag is removing the error, help text and adding tooltips

Reply:
Yeah makes sense